lib/screens/settings_screen.dart

```diff
@@ -256,11 +257,14 @@ class AppSettings extends InheritedWidget {
         _loadLegacy = preferences.getBool(_loadLegacyKey, defaultValue: true),
         isTestMode =
             const bool.fromEnvironment('testing_mode', defaultValue: false),
+        _showGuideOnStart =
+            preferences.getBool(_showGuideOnStartKey, defaultValue: true),
```

Comment:
i think it would be better to default this to false

Reply:
The user can deactivate it when it is shown, I think it would be better to show it at least once to the user.